packages/dds/tree/src/simple-tree/unhydratedFlexTreeFromInsertable.ts

```diff
@@ -355,8 +363,10 @@ function mapToFlexContent(data: FactoryContent, schema: TreeNodeSchema): FlexCon
  * @param data - The tree data to be transformed. Must be a Record-like object.
  * @param schema - The schema associated with the value.
  */
-function objectToFlexContent(data: FactoryContent, schema: TreeNodeSchema): FlexContent {
-	assert(isObjectNodeSchema(schema), 0x924 /* Expected an Object schema. */);
+function objectToFlexContent(
+	data: FactoryContent,
+	schema: ObjectNodeSchema & ObjectNodeSchemaInternalData,
```

Comment:
@CraigMacomber do we have a type alias for this?

Reply:
I don't think so. Based on new conventions, I think ObjectNodeSchemaPrivate would make sense for a name for ObjectNodeSchema but including non exported details.